api/peripheral_api/routes/Printer.js

```diff
@@ -9,19 +9,23 @@ const {
 const s3BucketKeys = require('../../config/config.json').S3Bucket;
 const printingS3Bucket = require('../../config/config.json').PrintingS3Bucket;
```

Comment:
```suggestion
const { s3BucketKeys, printingS3Bucket }  = require('../../config/config.json');
```

You can import multiple json properties all together like this